crates/p2p_proto/proto/common.proto

```diff
@@ -35,16 +43,17 @@ message InvokeTransaction {
 }
 
 message DeclareTransaction {
-  ContractClass contract_class = 1;
+  FieldElement contract_class_hash = 1;
```

Comment:
Internally we use the term "class" and "class hash" but in the p2p spec I like "contract class hash" better, it leaves no room for confusion.